utils/k8s/openebscrd.go

```diff
@@ -40,3 +40,26 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
 	spcClient := k8s.OpenebsClientSet.Openebs().StoragePoolClaims()
 	return spcClient.Delete(spcName, &meta_v1.DeleteOptions{})
 }
+
+// GetCStorVolumeReplica returns the CStorVolumeReplica object for given cvrName, namespace.
+// :return: *openebs_v1.CStorVolumeReplica: Pointer to CStorVolumeReplica objects.
+func (k8s K8S) GetCStorVolumeReplica(cvrName, namespace string) (*openebs_v1.CStorVolumeReplica, error) {
+	cvrClient := k8s.OpenebsClientSet.Openebs().CStorVolumeReplicas(namespace)
+	return cvrClient.Get(cvrName, meta_v1.GetOptions{})
+}
+
+// ListCStorVolumeReplica returns all the CStorVolumeReplica objects for given namespace
+func (k8s K8S) ListCStorVolumeReplica(namespace string) ([]openebs_v1.CStorVolumeReplica, error) {
```

Comment:
Can this be modified to return an encapsulating object like `CStorVolumeReplicaList` I see some other PRs are using this way.